src/net.c

```diff
@@ -1359,3 +1360,139 @@ void hyper_cleanup_dns(struct hyper_pod *pod)
 
 	close(fd);
 }
+
+int hyper_setup_iptables(struct hyper_pod *pod) {
+	int pid, status;
+	int pipefd[2] = {-1, -1};
+	int pipe_parent_err[2] = {-1, -1};
+	int ret = -1;
+	size_t size, offset = 0;
+	int pipe_size = 0;
+	const char *cmd = "iptables-restore";
```

Comment:
I avoided the #define , since this value is just used within this function and not anywhere else in the code.